wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -353,6 +353,7 @@ private void renderCombinedEventScripts()
 		}
 		if (combinedScript.length() > 0)
 		{
+			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_TIMERS_BOUND);");
```

Comment:
I'd be happy to change it to that.
Should we offer some level of backwards compat, and alias `AJAX_HANDLERS_BOUND`?